doc/source/skips/1-governance.rst

```diff
@@ -145,16 +146,18 @@ are made according to the following rules:
   sentence (but no change of the scikit-image.org landing page or the “about”
   page), require approval by a core developer *and* no disagreement or requested
   changes by a core developer on the issue or pull request page (lazy
-  consensus). Core developers are expected to give “reasonable time” to others
-  to give their opinion on the pull request if they’re not confident others
-  would agree.
+  consensus). Core developers are expected to wait one or two days for others to give
+  their opinion if they’re not confident that others would agree.
 
-- **Code changes and major documentation changes** require agreement by *two*
-  core developers *and* no disagreement or requested changes by a core developer
-  on the issue or pull-request page (lazy consensus).
+- **Code and major documentation changes, and changes to the API** require agreement by
+  *two* core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull request page (lazy consensus). In the case of disagreement or
+  requested changes, core developers are expected to wait about a week for others to
```

Comment:
"about a week" is confusing; perhaps rather "at least N days", with N being 5?

Reply:
I don't find it confusing personally, but I don't mind replacing it with something like "somewhere between 5 and 10 days." The problem I see with "at least N days" is that it provides only a lower bound. Then, the (implicit) upper bound becomes, in my understanding, by default, *6 months*, which is the time for the dormant bot to ping us back. I find it too long and unreasonable. I find it reasonable to wait "at least 5 days" (but not more than 10, 15, or 20). Let's make it clear again that this waiting period is understood *with no activity* -- if a fierce discussion is going on, obviously I'm not saying it should be interrupted after 10 days!

Reply:
Ok.